python_helpers/tx_udp_sender.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 Otis Elevator Company. All rights reserved.
+# Use of this source code is govered by the MIT license which
+# can be found in the LICENSE file.
+
+# Author: Jeremy Mill: jeremy.mill@otis.com
+
+# Otis udp_rx software has been designed to utilize information
+# security technology described Part 774 of the EAR Category 5 Part 2
```

Comment:
"described _in_" This issue might be in all of the files. If so don't worry about it.

Reply:
it is, we got that from lawyers, we can't change it